code/datums/traits/neutral.dm

```diff
@@ -257,15 +257,12 @@
 	var/mob/living/carbon/human/H = quirk_holder
 	var/obj/item/organ/tongue/old_tongue = locate() in H.internal_organs
 	var/obj/item/organ/tongue/tied/new_tongue = new(get_turf(H))
-	var/obj/item/clothing/gloves/old_gloves = locate() in H
 	var/obj/item/clothing/gloves/radio/gloves = new(get_turf(H))
 	old_tongue.Remove(H)
 	new_tongue.Insert(H)
 	qdel(old_tongue)
-	qdel(old_gloves)
```

Comment:
?

Reply:
that's hacky code i wrote that they don't need with their way of fixing it i think

Reply:
The code in there was so your glove slot got cleared up, so the force equip proc could equip the new gloves cleanly, this is not needed now, and you'll get both your old and new gloves, just like with nearsightness and the pair of glasses, or blindless and the blindfold.